packages/ui-react/src/components/Payment/Payment.tsx

```diff
@@ -219,78 +201,18 @@ const Payment = ({
-                  <Button
-                    className={styles.upgradeSubscription}
-                    label={t('user:payment.change_subscription')}
-                    disabled={!canRenewSubscription && activeSubscription.status === 'cancelled'}
-                    onClick={() => {
-                      if (offers.length > 1 && !canRenewSubscription) {
-                        setIsChangingOffer(true);
-                      } else {
-                        onUpgradeSubscriptionClick?.();
-                      }
-                    }}
-                    fullWidth={isMobile}
-                    color="primary"
-                    data-testid="change-subscription-button"
-                  />
-                )
-              )}
-              {(activeSubscription.status === 'active' || activeSubscription.status === 'active_trial') &&
-              !isGrantedSubscription &&
-              !isChangingOffer &&
-              canRenewSubscription ? (
-                <Button
-                  label={t('user:payment.cancel_subscription')}
-                  onClick={onCancelSubscriptionClick}
-                  fullWidth={isMobile}
-                  data-testid="cancel-subscription-button"
-                />
-              ) : canRenewSubscription ? (
-                <Button label={t('user:payment.renew_subscription')} onClick={onRenewSubscriptionClick} />
-              ) : null}
-            </React.Fragment>
-          ) : isLoading ? null : (
-            <React.Fragment>
-              <p>{t('user:payment.no_subscription')}</p>
-              <Button variant="contained" color="primary" label={t('user:payment.complete_subscription')} onClick={onCompleteSubscriptionClick} />
-            </React.Fragment>
-          )}
-          {isChangingOffer && (
-            <div className={styles.changePlanContainer}>
-              {offers
-                .filter((o) => o.planSwitchEnabled)
-                .map((offer) => (
-                  <OfferSwitch
-                    key={offer.offerId}
-                    isCurrentOffer={offer.offerId === activeSubscription?.accessFeeId}
-                    pendingDowngradeOfferId={pendingDowngradeOfferId}
-                    offer={offer}
-                    selected={selectedOfferId === offer.offerId}
-                    onChange={(offerId) => setSelectedOfferId(offerId)}
-                    expiresAt={activeSubscription?.expiresAt}
-                  />
-                ))}
-              <div className={styles.changePlanButtons}>
-                <Button
-                  label={t('user:account.save')}
-                  onClick={onChangePlanClick}
-                  disabled={selectedOfferId === activeSubscription?.accessFeeId || changeSubscriptionPlan.isLoading}
-                />
-                <Button label={t('user:account.cancel')} onClick={() => setIsChangingOffer(false)} variant="text" />
-                {activeSubscription?.status !== 'cancelled' && (
```

Comment:
This removes the following features for **all** integrations:

- Downgrade/upgrade subscription
- Renew subscription when it has been cancelled
- Complete subscription when user is not subscribed

I don't think we want to do that (yet)... @dbudzins @AntonLantukh ?

Reply:
I will close these pr i've opened new pr